README.md

```diff
@@ -77,6 +77,22 @@ See <https://docs.celestia.org/nodes/celestia-app> for more information.
 
 ## Usage
 
+First, make sure that the BBR congestion control algorithm is enabled in the
```

Comment:
[nit] a link would be helpful. Also expand the abbreviation b/c this is the first usage a reader will see

```suggestion
First, make sure that the [BBR](https://www.ietf.org/archive/id/draft-cardwell-iccrg-bbr-congestion-control-01.html) ("Bottleneck Bandwidth and Round-trip propagation time") congestion control algorithm is enabled in the
```